pom.xml

```diff
@@ -3,17 +3,18 @@
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<parent>
-		<groupId>life.qbic</groupId>
-		<artifactId>parent-pom</artifactId>
-		<version>2.2.0</version>
-	</parent>
+	<groupId>life.qbic</groupId>
 	<artifactId>sample-tracking-helper-lib</artifactId>
 	<version>1.0.4</version>
 	<name>Sample Tracking Helper API</name>
 	<url>http://github.com/qbicsoftware/sample-tracking-helper-lib</url>
 	<description>An easy to use interface providing sample tracking update functions for ETL scripts</description>
 	<packaging>jar</packaging>
+	<properties>
+		<java.version>8</java.version>
+		<maven.compiler.source>8</maven.compiler.source>
+		<maven.compiler.target>8</maven.compiler.target>
```

Comment:
this is 1.8 in data-model-lib, not sure if both work?

Reply:
the build runtime loads Java 8, so I think it should be fine